tests/unit/deploy/test_planner.py

```diff
@@ -284,6 +284,55 @@ def test_can_set_variables_when_needed(self):
         assert role_arn.name == 'myrole-dev_role_arn'
 
 
+class TestPlanS3Events(BasePlannerTests):
+    def test_can_plan_s3_event(self):
+        function = create_function_resource('function_name')
+        bucket_event = models.S3BucketNotification(
+            resource_name='function_name-s3event',
+            bucket='mybucket',
+            events=['s3:ObjectCreated:*'],
+            prefix=None,
+            suffix=None,
+            lambda_function=function,
+        )
+        plan = self.determine_plan(bucket_event)
+        self.assert_apicall_equals(
```

Comment:
My memory is a little fuzzy. Why do we have these separate `assert_apicall_equals` instead of just having a single `assert plan == expected_plan`?